{"name": "test01_add_member[userid重复，断言失败]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 60102 == 0", "trace": "self = <test_case.member.test_member.TestMember object at 0x00000157FCBC21C0>, userid = 'tong', name = 'tong4'\nmobile = '13222661162', errcode = 0, errmsg = 'created'\n\n    @pytest.mark.parametrize((\"userid,name,mobile,errcode,errmsg\"),add_data,ids=add_ids)\n    def test01_add_member(self,userid,name,mobile,errcode,errmsg):\n        log.info(\"-------开始测试增加成员-------\")\n    \n        res=self.member.add_member(self.token,userid,name,mobile)\n        log.info(f\"打印响应结果:{res}\" )\n        log.info(\"-------测试结束-------\")\n>       assert res[\"errcode\"] == errcode\nE       assert 60102 == 0\n\ntest_member.py:30: AssertionError"}, "attachments": [{"name": "log", "source": "532a87b6-3bf1-4ea3-9b68-b281eee108ab-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "715e5722-e3b2-4e0a-9cfc-65398a668700-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "userid", "value": "'tong'"}, {"name": "name", "value": "'tong4'"}, {"name": "mobile", "value": "'13222661162'"}, {"name": "errcode", "value": "0"}, {"name": "errmsg", "value": "'created'"}], "start": 1603977983523, "stop": 1603977984008, "uuid": "08879182-2104-48d5-ab36-8ad1f5b2ca89", "historyId": "3c8d1eedfb8e1f16669317667c9925df", "testCaseId": "a244cf98b03e2f32dd1c74ce8e22d393", "fullName": "test_case.member.test_member.TestMember#test01_add_member", "labels": [{"name": "parentSuite", "value": "test_case.member"}, {"name": "suite", "value": "test_member"}, {"name": "subSuite", "value": "TestMember"}, {"name": "host", "value": "LAPTOP-4F3KR3S3"}, {"name": "thread", "value": "3600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.member.test_member"}]}